avoidance/src/common.cpp

```diff
@@ -142,16 +142,16 @@ float getPitchFromQuaternion(const Eigen::Quaternionf q) {
   return pitch * RAD_TO_DEG;
 }
 
-void wrapAngleToPlusMinusPI(float& angle) {
-  angle = angle - 2.0f * M_PI_F * std::floor(angle / (2.0f * M_PI_F) + 0.5f);
+float wrapAngleToPlusMinusPI(float angle) {
```

Comment:
If we're going to change these, then we should add a 
```c++
#define WARN_UNUSED __attribute__((warn_unused_result))
```
```c++
float WARN_UNUSED wrapAngleToPlusMinusPI(float angle) {
```
In case anybody uses the old convention by mistake. And then we